### PR TITLE
feat: freetext command

### PR DESCRIPTION
**Pull request by Spherrrical**
Feature: .freetext

## Description

Added .freetext which helps new users learn how to use the Free Text in-game.

## Discord Username

```Spherical#0001```

### CODE REVIEW

File: src/commands/a32nx/freetext.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed } from '../../lib/embed';
+
+export const printer: CommandDefinition = {
```
Comment (oim1) [inline src/commands/a32nx/freetext.ts:5]:
```suggestion
export const freetext: CommandDefinition = {
```

File: src/commands/a32nx/freetext.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    name: 'freetext',
```
Comment (oim1) [inline src/commands/a32nx/freetext.ts:6]:
```suggestion
    name: [ 'freetext', 'ft' ],
```

**Review by oim1 — CHANGES_REQUESTED** [review]

File: src/commands/a32nx/freetext.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    description: 'Provides a link to the FlyByWire free text tutorial video',
+    category: CommandCategory.A32NX,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire A32NX | How do I use the Free Text feature?',
+        description: 'Please watch our video tutorial on how to use the FlyByWire A32NX Free Text feature [here.](https://www.youtube.com/watch?v=xED3mXqzxTc)',
```
Comment (benw202) [inline src/commands/a32nx/freetext.ts:11]:
```suggestion
        description: 'Please watch our [video tutorial](https://www.youtube.com/watch?v=xED3mXqzxTc) on how to use the FlyByWire A32NX Free Text feature'.
```

File: src/commands/a32nx/freetext.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed } from '../../lib/embed';
+
+export const printer: CommandDefinition = {
+    name: 'freetext',
+    description: 'Provides a link to the FlyByWire free text tutorial video',
+    category: CommandCategory.A32NX,
+    executor: (msg) => msg.channel.send(makeEmbed({
+        title: 'FlyByWire A32NX | How do I use the Free Text feature?',
```
Comment (benw202) [inline src/commands/a32nx/freetext.ts:10]:
```suggestion
        title: 'FlyByWire A32NX | Free Text',
```

**Review by benw202 — CHANGES_REQUESTED** [review]

File: src/commands/a32nx/freetext.ts (hunk 3 above)
Comment (benw202) [reply src/commands/a32nx/freetext.ts:11]: Replaced YouTube link with docs link as the YouTube video is no longer current.

Comment (Valastiri) [reply src/commands/a32nx/freetext.ts:11]:
```suggestion
        description: 'Please watch our video tutorial on how to use the FlyByWire A32NX Free Text feature [here.](https://docs.flybywiresim.com/fbw-a32nx/feature-guides/freetext)',
```
Please use this link instead as we are / have created a dedicated feature page for this on docs. 

Relevant PR - https://github.com/flybywiresim/docs/pull/284

**Pull request closed without merging**